### PR TITLE
docs: update migration-v5

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update migration-v5 |
| 🇨🇳 Chinese | docs: update migration-v5 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/migration-v5.zh-CN.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ title: 从 v4 到 v5
 - 弃用 less，采用 CSS-in-JS，更好地支持动态主题。底层使用 [@ant-design/cssinjs](https://github.com/ant-design/cssinjs) 作为解决方案。
   - 所有 less 文件全部移除，less 变量不再支持透出。
   - 产物中不再包含 css 文件。由于 CSS-in-JS 支持按需引入，原本的 `antd/dist/antd.css` 也已经移除，如果需要重置一些基本样式请引入 `antd/dist/reset.css`。
+  - 如果需要获取基于 `.ant-app` 的默认重置样式，可以使用[App 包裹组件](components/app-cn)，解决原生元素没有 antd 规范样式的问题。
```
Comment (afc163) [inline docs/react/migration-v5.zh-CN.md:27]: > 获取基于 `.ant-app` 的默认重置样式

这个感觉不是需求的描述。

Comment (li-jia-nan) [reply docs/react/migration-v5.zh-CN.md:27]: `如果需要获取默认重置样式` 这样可以吗？

Comment (afc163) [reply docs/react/migration-v5.zh-CN.md:27]: 如果需要组件重置样式，又不想用 `antd/dist/reset.css` 从而导致污染全局样式的话。可以尝 ****

Comment (li-jia-nan) [reply docs/react/migration-v5.zh-CN.md]: done~

File: docs/react/migration-v5.zh-CN.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ title: 从 v4 到 v5
 - 弃用 less，采用 CSS-in-JS，更好地支持动态主题。底层使用 [@ant-design/cssinjs](https://github.com/ant-design/cssinjs) 作为解决方案。
   - 所有 less 文件全部移除，less 变量不再支持透出。
   - 产物中不再包含 css 文件。由于 CSS-in-JS 支持按需引入，原本的 `antd/dist/antd.css` 也已经移除，如果需要重置一些基本样式请引入 `antd/dist/reset.css`。
-  - 如果需要获取基于 `.ant-app` 的默认重置样式，可以使用[App 包裹组件](components/app-cn)，解决原生元素没有 antd 规范样式的问题。
+  - 如果需要组件重置样式，又不想引入 `antd/dist/reset.css` 从而导致污染全局样式的话。可以尝试使用[App 包裹组件](components/app-cn)，解决原生元素没有 antd 规范样式的问题。
```
Comment (afc163) [inline docs/react/migration-v5.zh-CN.md]:
```suggestion
  - 如果需要组件重置样式，又不想引入 `antd/dist/reset.css` 从而导致污染全局样式的话。可以尝试在应用最外层使用[App 组件](components/app-cn)，解决原生元素没有 antd 规范样式的问题。
```

Comment (li-jia-nan) [reply docs/react/migration-v5.zh-CN.md]: done~

**Review by afc163 — APPROVED** [review]

**Pull request merged**